### ISSUE TITLE
Testing issue

### ISSUE THREAD

**Issue by hayd** [jpfairbanks/GraphMatrices.jl#13]
I think this is partially due to FactCheck.... I think the 0.5 is something bad on my install (from source), but I'm not sure.

Strange that I'd see this when it passes travis.

0.4-rc-1

```
julia> Pkg.test("GraphMatrices")
INFO: Testing GraphMatrices
INFO: Recompiling stale cache file /Users/andy/.julia/lib/v0.4/Compat.ji for module Compat.
WARNING: module GraphMatrices should explicitly import .* from Base
WARNING: module GraphMatrices should explicitly import == from Base
WARNING: module GraphMatrices should explicitly import * from Base
WARNING: The `=>` syntax is deprecated, use `-->` instead
constructors
  > Adjacency
    Error :: (line:505) :: Adjacency
      Expression: convert(SparseMatrix{Float64},stochmat) --> truthy
      UndefVarError: truthy not defined
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:259
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in do_fact at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:333
       [inlined code] from /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in test_adjacency at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:26
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:153
       in context at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:474
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:152
       in facts at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:448
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in process_options at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in _start at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
    Error :: (line:505) :: Adjacency
      Expression: convert(SparseMatrix{Float64},adjhat) --> truthy
      UndefVarError: truthy not defined
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:259
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in do_fact at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:333
       [inlined code] from /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in test_adjacency at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:26
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:153
       in context at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:474
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:152
       in facts at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:448
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in process_options at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in _start at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
    Error :: (line:505) :: Adjacency
      Expression: convert(SparseMatrix{Float64},avgmat) --> truthy
      UndefVarError: truthy not defined
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:259
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in do_fact at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:333
       [inlined code] from /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in test_adjacency at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:26
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:153
       in context at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:474
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:152
       in facts at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:448
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in process_options at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in _start at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
  > Laplacian
    Error :: (line:505) :: Laplacian
      Expression: lapl --> truthy
      UndefVarError: truthy not defined
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:259
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in do_fact at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:333
       [inlined code] from /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in test_laplacian at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:40
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:157
       in context at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:474
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:156
       in facts at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:448
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in process_options at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in _start at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
    Error :: (line:505) :: Laplacian
      Expression: StochasticAdjacency(adjacency(lapl)) --> truthy
      UndefVarError: truthy not defined
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:259
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in do_fact at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:333
       [inlined code] from /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in test_laplacian at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:40
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:157
       in context at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:474
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:156
       in facts at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:448
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in process_options at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in _start at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
    Error :: (line:505) :: Laplacian
      Expression: NormalizedAdjacency(adjacency(lapl)) --> truthy
      UndefVarError: truthy not defined
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:259
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in do_fact at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:333
       [inlined code] from /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in test_laplacian at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:40
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:157
       in context at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:474
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:156
       in facts at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:448
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in process_options at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in _start at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
    Error :: (line:505) :: Laplacian
      Expression: AveragingAdjacency(adjacency(lapl)) --> truthy
      UndefVarError: truthy not defined
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:259
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in do_fact at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:333
       [inlined code] from /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in test_laplacian at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:40
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:157
       in context at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:474
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:156
       in facts at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:448
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in process_options at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in _start at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
    Error :: (line:505) :: Laplacian
      Expression: adjacency(lapl) --> truthy
      UndefVarError: truthy not defined
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:259
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in do_fact at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:333
       [inlined code] from /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in test_laplacian at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:40
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:157
       in context at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:474
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:156
       in facts at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:448
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in process_options at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in _start at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
    Error :: (line:505) :: Laplacian
      Expression: sparse(lapl) --> truthy
      UndefVarError: truthy not defined
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:259
       in anonymous at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in do_fact at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:333
       [inlined code] from /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:271
       in test_laplacian at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:40
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:157
       in context at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:474
       in anonymous at /Users/andy/.julia/v0.4/GraphMatrices/test/types.jl:156
       in facts at /Users/andy/.julia/v0.4/FactCheck/src/FactCheck.jl:448
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in include_from_node1 at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in process_options at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
       in _start at /Applications/Julia-0.4.0-rc1.app/Contents/Resources/julia/lib/julia/sys.dylib
  > Accessors
Out of 43 total facts:
  Verified: 34
  Errored:  9
arithmetic
adjmat * ones(Float64,n) = [9.0,3.0,6.0,8.0,5.0,6.0,6.0,6.0,3.0,5.0]
12 facts verified.
other tests
  > equality testing 
20 facts verified.
Random Walk Demo
1 fact verified.
INFO: GraphMatrices tests passed
```

0.5-dev+592 (from source)

```
julia> Pkg.test("GraphMatrices")
INFO: Testing GraphMatrices
constructors
  > Adjacency
  > Laplacian
    Error :: (line:505) :: Laplacian
      Expression: convert(Adjacency,lapl) --> truthy
      UndefVarError: Adjacency not defined
       in anonymous at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:271
       in do_fact at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:333
       [inlined code] from /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:271
       in test_laplacian at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:258
       in anonymous at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:167
       in context at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:474
       in anonymous at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:166
       in facts at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:448
       in include at ./boot.jl:261
       in include_from_node1 at ./loading.jl:304
       in include at ./boot.jl:261
       in include_from_node1 at ./loading.jl:304
       in process_options at ./client.jl:275
       in _start at ./client.jl:375
  > Accessors
Out of 43 total facts:
  Verified: 42
  Errored:  1
arithmetic
adjmat * ones(Float64,n) = [3.0,2.0,5.0,3.0,2.0,2.0,6.0,5.0,4.0,4.0]
  Error :: (line:505)
    Expression: ((eigs(adjmat,which=:LR))[1])[1] --> greater_than(1.0)
    error compiling saupd: could not load library "libarpack"
    dlopen(libarpack.dylib, 1): Library not loaded: @rpath/libopenblas.dylib
      Referenced from: /Users/andy/projects/julia/usr/lib//libarpack.dylib
      Reason: image not found
     in aupd_wrapper at linalg/arpack.jl:49
     in eigs at linalg/arnoldi.jl:215
     in eigs at linalg/arnoldi.jl:50
     in anonymous at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:271
     in do_fact at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:333
     [inlined code] from /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:271
     in test_arithmetic at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:258
     in anonymous at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:179
     in facts at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:448
     in include at ./boot.jl:261
     in include_from_node1 at ./loading.jl:304
     in include at ./boot.jl:261
     in include_from_node1 at ./loading.jl:304
     in process_options at ./client.jl:275
     in _start at ./client.jl:375
  Error :: (line:505)
    Expression: ((eigs(stochmat,which=:LR))[1])[1] --> roughly(1.0)
    error compiling naupd: could not load library "libarpack"
    dlopen(libarpack.dylib, 1): Library not loaded: @rpath/libopenblas.dylib
      Referenced from: /Users/andy/projects/julia/usr/lib//libarpack.dylib
      Reason: image not found
     in aupd_wrapper at linalg/arpack.jl:49
     in eigs at linalg/arnoldi.jl:215
     in eigs at linalg/arnoldi.jl:50
     in anonymous at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:271
     in do_fact at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:333
     [inlined code] from /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:271
     in test_arithmetic at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:258
     in anonymous at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:179
     in facts at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:448
     in include at ./boot.jl:261
     in include_from_node1 at ./loading.jl:304
     in include at ./boot.jl:261
     in include_from_node1 at ./loading.jl:304
     in process_options at ./client.jl:275
     in _start at ./client.jl:375
  Error :: (line:505)
    Expression: ((eigs(avgmat,which=:LR))[1])[1] --> roughly(1.0)
    error compiling naupd: could not load library "libarpack"
    dlopen(libarpack.dylib, 1): Library not loaded: @rpath/libopenblas.dylib
      Referenced from: /Users/andy/projects/julia/usr/lib//libarpack.dylib
      Reason: image not found
     in aupd_wrapper at linalg/arpack.jl:49
     in eigs at linalg/arnoldi.jl:215
     in eigs at linalg/arnoldi.jl:50
     in anonymous at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:271
     in do_fact at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:333
     [inlined code] from /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:271
     in test_arithmetic at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:258
     in anonymous at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:179
     in facts at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:448
     in include at ./boot.jl:261
     in include_from_node1 at ./loading.jl:304
     in include at ./boot.jl:261
     in include_from_node1 at ./loading.jl:304
     in process_options at ./client.jl:275
     in _start at ./client.jl:375
  Error :: (line:505)
    Expression: ((eigs(lapl,which=:LR))[1])[1] --> greater_than(2.0)
    error compiling saupd: could not load library "libarpack"
    dlopen(libarpack.dylib, 1): Library not loaded: @rpath/libopenblas.dylib
      Referenced from: /Users/andy/projects/julia/usr/lib//libarpack.dylib
      Reason: image not found
     in aupd_wrapper at linalg/arpack.jl:49
     in eigs at linalg/arnoldi.jl:215
     in eigs at linalg/arnoldi.jl:50
     in anonymous at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:271
     in do_fact at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:333
     [inlined code] from /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:271
     in test_arithmetic at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:258
     in anonymous at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:179
     in facts at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:448
     in include at ./boot.jl:261
     in include_from_node1 at ./loading.jl:304
     in include at ./boot.jl:261
     in include_from_node1 at ./loading.jl:304
     in process_options at ./client.jl:275
     in _start at ./client.jl:375
  Error :: (line:505)
    Expression: ((eigs(lhat,which=:LR))[1])[1] --> less_than(2.0 + 1.0e-9)
    error compiling saupd: could not load library "libarpack"
    dlopen(libarpack.dylib, 1): Library not loaded: @rpath/libopenblas.dylib
      Referenced from: /Users/andy/projects/julia/usr/lib//libarpack.dylib
      Reason: image not found
     in aupd_wrapper at linalg/arpack.jl:49
     in eigs at linalg/arnoldi.jl:215
     in eigs at linalg/arnoldi.jl:50
     in anonymous at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:271
     in do_fact at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:333
     [inlined code] from /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:271
     in test_arithmetic at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:258
     in anonymous at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:179
     in facts at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:448
     in include at ./boot.jl:261
     in include_from_node1 at ./loading.jl:304
     in include at ./boot.jl:261
     in include_from_node1 at ./loading.jl:304
     in process_options at ./client.jl:275
     in _start at ./client.jl:375
Out of 12 total facts:
  Verified: 7
  Errored:  5
other tests
  > equality testing
ERROR: LoadError: LoadError: error compiling saupd: could not load library "libarpack"
dlopen(libarpack.dylib, 1): Library not loaded: @rpath/libopenblas.dylib
  Referenced from: /Users/andy/projects/julia/usr/lib//libarpack.dylib
  Reason: image not found
 in aupd_wrapper at linalg/arpack.jl:49
 in eigs at linalg/arnoldi.jl:215
 in eigs at linalg/arnoldi.jl:50
 in test_punchedmatrix at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:154
 in anonymous at /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl:187
 in facts at /Users/andy/.julia/v0.5/FactCheck/src/FactCheck.jl:448
 in include at ./boot.jl:261
 in include_from_node1 at ./loading.jl:304
 in include at ./boot.jl:261
 in include_from_node1 at ./loading.jl:304
 in process_options at ./client.jl:275
 in _start at ./client.jl:375
while loading /Users/andy/.julia/v0.5/GraphMatrices/test/types.jl, in expression starting on line 182
while loading /Users/andy/.julia/v0.5/GraphMatrices/test/runtests.jl, in expression starting on line 5
================================[ ERROR: GraphMatrices ]=================================

failed process: Process(`/Users/andy/projects/julia/usr/bin/julia --check-bounds=yes --code-coverage=none --color=yes /Users/andy/.julia/v0.5/GraphMatrices/test/runtests.jl`, ProcessExited(1)) [1]

=========================================================================================
ERROR: Base.Pkg.PkgError("GraphMatrices had test errors")
 in test at pkg/entry.jl:832
 in anonymous at pkg/dir.jl:31
 in cd at file.jl:22
 in cd at pkg/dir.jl:31
 in test at pkg.jl:75
```

**Comment by jpfairbanks** [comment]
For the julia v0.4 testing errors:

What version of GraphMatrices do you have? Line 8 https://github.com/jpfairbanks/GraphMatrices.jl/blob/d336fb0d98bc013c8fb76c4c3b92b11212e51d26/test/types.jl#L8 has a definition of truthy and I fixed the => to --> warning a while ago. 
master should not give those errors/warnings.

For the julia v0.5 errors:

```
Expression: ((eigs(adjmat,which=:LR))[1])[1] --> greater_than(1.0)
    error compiling saupd: could not load library "libarpack"
    dlopen(libarpack.dylib, 1): Library not loaded: @rpath/libopenblas.dylib
      Referenced from: /Users/andy/projects/julia/usr/lib//libarpack.dylib
```

means that julia can't find your openblas dynamic library. Did you change any of the loadpaths in the makefile for installing julia v0.5? Either arpack/blas did not get built correctly or julia can't find it. GraphMatrices just uses `Base.eigs` so my guess is that it is a problem upstream. To confirm that you can call `a = sprand(10,10,0.3); eigs(a)` and look for errors. If `eigs` fails on a `Base.SparseMatrixCSC`, it is a problem upstream of GraphMatrices.

**Comment by hayd** [comment]
fast forwarding to master fixed my 0.4. I guess I screwed up installing from source somehow.

```
julia> a = sprand(10,10,0.3); eigs(a)
ERROR: error compiling naupd: could not load library "libarpack"
dlopen(libarpack.dylib, 1): Library not loaded: @rpath/libopenblas.dylib
  Referenced from: /Users/andy/projects/julia/usr/lib//libarpack.dylib
  Reason: image not found
 in aupd_wrapper at linalg/arpack.jl:49
 in eigs at linalg/arnoldi.jl:215
 in eigs at linalg/arnoldi.jl:50
```

Thanks!
